core/src/main/scala/org/bitcoins/core/util/TimeUtil.scala

```diff
@@ -13,4 +17,25 @@ object TimeUtil {
   def currentEpochSecond: Long = {
     now.getEpochSecond
   }
+
+  def iso8601ToDate(str: String): Date = {
```

Comment:
What was the reason for using `Date` rather than `Instant`? It seems `Instant` is superior in every way. Was there some compatibility issue?

Reply:
We can't use `Instant` because we pass that back and forth using epoch second. To send it using a different format we need to use a different type.